unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/Passport/Passport/PlayerPassportHUDView.cs

```diff
@@ -0,0 +1,40 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.UI;
+using UnityEngine.Events;
+
+public class PlayerPassportHUDView : MonoBehaviour
+{
+    [SerializeField] private PassportPlayerInfoComponentView playerInfoView;
```

Comment:
did you try doing for example: `[field: SerializeField] private PassportPlayerInfoComponentView PlayerInfoView { get; private set; }`

Reply:
With that the result is
`Assets/Scripts/MainScripts/DCL/Controllers/HUD/Passport/Passport/PlayerPassportHUDView.cs(10,99): error CS0273: The accessibility modifier of the 'PlayerPassportHUDView.PlayerInfoView.set' accessor must be more restrictive than the property or indexer 'PlayerPassportHUDView.PlayerInfoView'
`

It would force me to set the field as public, but then it goes against the entire idea xD